dist/rw-widgets.js

```diff
@@ -122,6 +122,10 @@ CrisisOverviewWidget.prototype.compile = function(elements, next) {
   var config = this.config();
   this.config('adjustedTitle', titleAdjust(config.title));
 
+  if (config.environment && config.map.src) {
+    this.config('map.src', config.environment.content + config.map.src);
```

Comment:
This doesn't seem to be used below?

Also, what happens if the either of these is undefined? Could we throw a console warning?